lib/Service/AttachmentService.php

```diff
@@ -80,43 +83,50 @@ public function __construct(IRootFolder $rootFolder,
 	 * @throws \OCP\Files\InvalidPathException
 	 * @throws \OCP\Files\NotPermittedException
 	 */
```

Comment:
Nitpicking, but seems like you forgot to update the annotations here.

Reply:
:+1: